aws-py-voting-app/blogpost.md

```diff
@@ -105,15 +106,4 @@ exec_policy_attachment = aws.iam.RolePolicyAttachment("app-exec-policy", role=ap
 
 ---
 
-The full code for the blog post can be [found on github.](https://github.com/jetvova/examples/tree/vova/aws-py-flask-redis-voting-app/aws-py-voting-app)
-
-
-
-
-
-
-
-
-
-
-
+The full code for the blog post can be [found on Github.](https://github.com/jetvova/examples/tree/vova/aws-py-flask-redis-voting-app/aws-py-voting-app)
```

Comment:
Add a conclusion along the lines of:
"In this example, I showed how easy it is to swap out infrastructure defined in TypeScript with one defined in Python using Pulumi. Teams often choose to standardize on one language and Pulumi lets them further standardize  on one language for frontend, backend and infrastructure code. 

Next week, I'll show how to further modernize the frontend by shifting form Flask to Django for the frontend and swap out Redis for Amazon RDS -- using Pulumi to stand up new resources as I go.